proto/config.proto

```diff
@@ -94,3 +95,22 @@ message ZoneConfig {
   // in the zone config hierarchy, up to the default policy if necessary.
   optional GCPolicy gc = 4 [(gogoproto.customname) = "GC", (gogoproto.moretags) = "yaml:\"gc,omitempty\""];
 }
+
+// RangeTreeRoot holds the root node and size of the range tree.
+message RangeTree {
+  optional int64 root_id = 1 [(gogoproto.nullable) = true, (gogoproto.customname) = "RootID"];
+  optional int64 count = 2 [(gogoproto.nullable) = false, (gogoproto.customname) = "Count"];
```

Comment:
Yeah, I actually had removed these, but they seem to have returned.  I'll take them out again.